packages/components/src/internal/components/domainproperties/assay/AssayDesignerPanels.tsx

```diff
@@ -235,6 +235,7 @@ class AssayDesignerPanelsImpl extends React.PureComponent<Props & InjectedBaseDo
                             useTheme={useTheme}
                             appPropertiesOnly={appPropertiesOnly}
                             showFilePropertyType={showFilePropertyType}
+                            showStudyPropertyTypes={!appPropertiesOnly}
```

Comment:
I think we need a comment here since using `appPropertiesOnly` for `showStudyPropertyTypes` is a little indirect. I believe the case here is that we don't want to show the study properties in the assay designer for LKSM app. `appPropertiesOnly` is set to false for LKB app so this would mean they would show there, is that desired?

Reply:
The `appPropertiesOnly` parameter needs perhaps a new name because of the different treatment in the different apps.  The generic name is bound to lead to confusion.